docs/stellar-core_example.cfg

```diff
@@ -101,6 +101,11 @@ KNOWN_PEERS=[
 "core-testnet2.stellar.org", 
 "core-testnet3.stellar.org"]
 
+# BOOTSTRAP_DNS_SRV (single string) default is '_stellar-core._tcp.stellar.org'
```

Comment:
I don't think this should default to anything. It basically defaults to something if we put this in the example config. If you are running standalone you don't want it to go fetch a bunch of peers from us.